app/src/main/java/com/example/coinage/fragments/SpendingLimitChartsFragment.java

```diff
@@ -64,7 +72,10 @@ public void done(List<SpendingLimit> spendingLimits, ParseException e) {
                     BigDecimal spendingLimitAmount = BigDecimal.valueOf(spendingLimit.getAmount().floatValue());
 
                     ParseQuery<Transaction> transactionQuery = ParseQuery.getQuery(Transaction.class);
-                    transactionQuery.whereEqualTo(Transaction.KEY_CATEGORY, category);
+                    if (!category.equals("Overall")) {
```

Comment:
Just like you've done with `Transaction.KEY_CATEGORY`, it's good to take hardcoded strings in code and put them as constants in one place. Maybe this one could be `Transaction.CATEGORY_OVERALL`?